package.json

```diff
@@ -10,10 +10,10 @@
   "scripts": {
     "dev": "node server.mjs",
     "build": "next build --no-lint",
-    "start": "node server.mjs",
     "lint": "next lint",
     "debug": "node --inspect server.mjs",
-    "server": "node server.mjs"
+    "start": "cross-env NODE_ENV='production' node server.mjs",
+    "postinstall": "cross-env NODE_ENV='production' npm run build"
```

Comment:
Can we add a script here called `dev-install` (or something similar) that runs `npm install --ignore-script`? This will allow us to install stuff and ignore the postinstall.

Reply:
done